packages/server/src/appMigrations/migrations/20250729134531_workspace_cleanups.ts

```diff
@@ -0,0 +1,32 @@
+import { context } from "@budibase/backend-core"
+import sdk from "../../sdk"
+import { Document } from "@budibase/types"
+
+const migration = async () => {
+  const allWorkspaceApps = await sdk.workspaceApps.fetch()
+  const defaultWorkspaceApps = allWorkspaceApps.filter(a => a.isDefault)
+  if (defaultWorkspaceApps.length > 1) {
+    const [workspaceAppToUse, ...toDelete] = defaultWorkspaceApps.sort((a, b) =>
```

Comment:
Forgive me, I haven't followed this work super closely. Is there a possibility that the user has modified one of the duplicate workspace apps and this selection might not choose the correct one? Should we perhaps select by the updatedAt field?